sspl_test/alerts/realstor/test_real_stor_fan_actuator.py

```diff
@@ -26,7 +26,7 @@ def test_real_stor_fan_module_actuator(agrs):
     instance_id = "4"
     resource_type = "enclosure:hw:fan"
     send_enclosure_request("ENCL:%s" % resource_type, instance_id)
-    ingressMsg = get_fru_response(resource_type, instance_id)
+    ingressMsg = get_fru_response(resource_type, "Fan Module 4")
```

Comment:
change on line 26 only

Reply:
instance_id passed in request and resource_id retrieved from response are not coming same.
Instance id represents resource index(instance) and resource_id represents resource specific.
get_fru_response() -- applies filter on the response for the matching resource_id.